app/views/placements/wizards/add_user_wizard/_user_step.html.erb

```diff
@@ -1,12 +1,11 @@
-<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
-<% content_for :page_title, user_step.errors.any? ? t(".title_with_error", caption:) : t(".title", caption:) %>
+<% content_for :page_title, user_step.errors.any? ? t(".title_with_error", caption: t(".caption")) : t(".title", caption: t(".caption")) %>
```

Comment:
Okay 🫠 well this is definitely not looking fun. Three layers deep of translation nesting is a bit much for my mere mortal brain.

Again, probably not something for this PR. But I wonder if we need to think about if/how this could become easier to understand.